eventsource/README.md

```diff
@@ -1,54 +1,2 @@
-# Server-Sent Events Test Collection
-
-Server-Sent Events [latest draft](http://dev.w3.org/html5/eventsource/).
-
-Following up work done during the TestTWF 2012 Paris event:
-
-Most tests comes from [Opera](http://tc.labs.opera.com/apis/EventSource/), are from august 2010 and probably only valid against [spec rev. ~1.139](http://dev.w3.org/cvsweb/~checkout~/html5/eventsource/Overview.html?rev=1.139;content-type=text%2Fhtml). You can check the following diff :
-
-[diff between 1.139 (23 Jul 2010) and 1.229 (25 Oct. 2012) revisions](http://dev.w3.org/cvsweb/html5/eventsource/Overview.html.diff?r1=text&tr1=1.139&r2=text&tr2=1.229)
-
-to get an idea of what needs to get updated.
-
-##DONE (updated against rev. 1.229):
-- **eventsource-constructor-url-bogus.htm**: whatwg r6602: renamed SYNTAX_ERR to SyntaxError
-
-- **eventsource-constructor-stringify.htm**: still valid. bugfix.
-
-##TODO (need to be updated against rev. 1.229):
```

Comment:
Confirmed that this was last updated ages ago, and that most tests have changed since then, so going through the TODOs is unlikely to be useful. Better to investigate failing tests.